recipes/nim/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# based on https://github.com/bioconda/bioconda-recipes/tree/master/recipes/pb-falconc
+
+set -vexu -o pipefail
+
+pushd nim
+# inject compilers
+echo "gcc.exe = \"$CC\"" >> config/nim.cfg
+echo "gcc.linkerexe = \"$CC\"" >> config/nim.cfg
+echo "clang.exe = \"$CC\"" >> config/nim.cfg
+echo "clang.linkerexe = \"$CC\"" >> config/nim.cfg
+./build.sh
+bin/nim c koch
+./koch tools
+
+ls -larth
```

Comment:
These `ls` commands shouldn't be needed.

Reply:
I think they are OK for debugging though

Reply:
Good. I will delete them in a future PR. I plan to add OSX and Win support later anyway.